app/api/v2/models.py

```diff
@@ -1,12 +1,13 @@
-from ...db_config import init_db
+import os
+from .dbmodel import SenditDb
+
+uri = os.getenv('DEV_DB_URI')
+SenditDb.start_db(uri)
```

Comment:
You're doing `start_db` twice. In `create_app ()` and here. Use a single instance